var/spack/repos/builtin/packages/py-jupyterhub/package.py

```diff
@@ -10,22 +10,26 @@ class PyJupyterhub(PythonPackage):
     """Multi-user server for Jupyter notebooks."""
 
     homepage = "https://pypi.org/project/jupyterhub"
-    url      = "https://pypi.io/packages/source/j/jupyterhub/jupyterhub-0.9.4.tar.gz"
+    url      = "https://pypi.io/packages/source/j/jupyterhub/jupyterhub-1.0.0.tar.gz"
 
-    version('0.9.4',    sha256='86b1cce446d4e8347e26913878858fc8964d103fde19b606fe37ccc5188d629d')
+    version('1.0.0',    sha256='33541a515a041b9a518ca057c1c4ab4215a7450fdddc206401713ee8137fa67f')
+    version('0.9.4',    sha256='7848bbb299536641a59eb1977ec3c7c95d931bace4a2803d7e9b28b9256714da')
 
-    depends_on('python@3.5:')
-    depends_on('node-js', type=('build', 'run'))
-    depends_on('py-setuptools', type='build')
-    depends_on('py-python-dateutil', type='run')
-    depends_on('py-jinja2', type='run')
-    depends_on('py-sqlalchemy', type='run')
-    depends_on('py-tornado', type='run')
-    depends_on('py-traitlets', type='run')
-    depends_on('py-alembic', type='run')
-    depends_on('py-mako', type='run')
-    depends_on('py-async-generator', type='run')
-    depends_on('py-jupyter-notebook', type='run')
-    depends_on('py-prometheus-client', type='run')
-    depends_on('py-send2trash', type='run')
-    depends_on('py-requests', type='run')
+    depends_on('python@3.5:', type=('build', 'run'))
+    depends_on('py-setuptools', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-setuptools', type='build')
```

Reply:
I had it set that way originally but when py-setuptools is build only I get an error like:
wspear@yu:~/bin/spack$ jupyterhub-singleuser 
Traceback (most recent call last):
  File "/home/wspear/bin/spack/opt/spack/linux-ubuntu19.04-westmere/gcc-8.3.0/py-jupyterhub-1.0.0-nn3adzsylbtppwtjqq53aug5aun3hdqr/bin/jupyterhub-singleuser", line 7, in <module>
    from pkg_resources import load_entry_point
ModuleNotFoundError: No module named 'pkg_resources'

If I manually activate py-setuptools (when it isn't set as a run dependency) then the error goes away.

Reply:
Huh, when I grep the tarball there are no imports of pkg_resources. I guess that script is auto-generated in some way. Thanks for checking!